src/Illuminate/Contracts/Auth/Guard.php

```diff
@@ -18,6 +18,13 @@ public function check();
      */
     public function guest();
 
+    /**
+     * Forget the current user.
+     *
+     * @return $this
+     */
+    public function forget();
```

Comment:
We can't add this method on the contract on a stable release because it's a breaking change. I wouldn't even add it on the next major version. Just add the method to the RequestGuard.

Reply:
What about not adding the method to the Contract but still in the trait? Because other guards are also affected by this issue, adding it to only the RequestGuard doesn't feel right